common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -209,7 +215,7 @@ function (VideoPlayer, i18n, moment) {
         }
     }
 
-    function _loadYoutubeApi(state) {
+    function _waitForYoutubeApi(state) {
         console.log('[Video info]: YouTube API is not loaded. Will try to load...');
```

Comment:
It seems that a lot of what is being logged here is at info or debug level. I don't know of a convention for declaring the log level client-side, but to me I like to see a clean console when I visit pages. Can this be enabled with a debug flag or something?